source/upgrade-guide/upgrading-central-components.rst

```diff
@@ -1,41 +1,42 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-   :description: Learn how to upgrade the Wazuh indexer, server, and dashboard to the latest version available.
+   :description: Learn how to upgrade the Wazuh central components, including the indexer, server, and dashboard, for all-in-one and multi-node deployments.
 
 Wazuh central components
 ========================
 
 This section guides you through the upgrade process of the Wazuh indexer, the Wazuh server, and the Wazuh dashboard.
 
-.. note:: You need root user privileges to run all the commands described below.
+- **All-in-one deployments:** Execute all commands and configuration actions on the same node since all components run on a single system.
+
+- **Multi-node cluster deployments:** Run commands and apply configurations on the respective node where the component being upgraded is located.
```

Comment:
```suggestion
-  **All-in-one deployments:** Execute all commands and configuration actions on the same node since all components run on a single system.
-  **Multi-node cluster deployments:** Run commands and apply configurations on the respective node where the component being upgraded is located.
```